springboot/README.md

```diff
@@ -100,9 +118,44 @@ Latency for GraalVM version:
   <img src="../imgs/springboot/springboot-native-log-insights.JPG" alt="GraalVM Version Log Insights"/>
 </p>
 
+Latency for SnapStart version:
+AWS Lambda service logs Restoration time differently when compared to cold start times in CloudWatch logs. 
+For this reason, we need different log insights queries to capture performance metrics for SnapStart functions. 
+Also, it's easy to get Cold Start and Warm Start performance metrics with two different queries rather than one. 
+
+Use the below query to get Cold start metrics for with SnapStart Lambda functions:
+
+```
+filter @message like "REPORT"
+| filter @message not like "RESTORE_REPORT"
+| filter @message like "Restore Duration"
+| parse @message "Restore Duration:* ms" as restoreTime
+| fields @duration + restoreTime as duration
+| stats count(*) as count, pct(duration, 50) as p50, pct(duration, 90) as p90, pct(duration, 99) as p99, max(duration) as max
+```
+
+<p align="center">
+  <img src="../imgs/springboot/springboot-snapstart-cold-log-insights.JPG" alt="Cold Start metrics with SnapStart"/>
+</p>
+
+Use the below query to get Warm start metrics for with SnapStart Lambda functions:
+```
+filter @message like "REPORT"
+| filter @message not like "RESTORE_REPORT"
+| filter @message not like "Restore Duration"
+| fields @duration as duration
+| stats count(*) as count, pct(duration, 50) as p50, pct(duration, 90) as p90, pct(duration, 99) as p99, max(duration) as max
+```
+
+<p align="center">
+  <img src="../imgs/springboot/springboot-snapstart-warm-log-insights.JPG" alt="Cold Start metrics with SnapStart"/>
+</p>
+
 ## AWS X-Ray Tracing
 You can add additional detail to your X-Ray tracing by adding a TracingInterceptor to your AWS SDK clients.
 
+Please note that AWS Lambda SnapStart currently does not support X-ray tracing. For this reason, tracing is disabled for all lambda functions in SnapStart version.
```

Comment:
Also mention which AWS regions support SnapStart today. Point to the user guide page, if available

Reply:
Added regions